service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/cafcass/CafcassCasesControllerTest.java

```diff
@@ -26,34 +56,80 @@
 public class CafcassCasesControllerTest extends AbstractTest {
```

Comment:
1. Remove the `Deprecated` annotation if you wanna reuse this file.
1. Also, maybe rename this test to something like CafcassApiDocumentUploadControllerTest, as it only covers upload document.

FYI, there is a CafcassApiControllerBaseTest which provide some basic stuff for Cafcass API controller unit test. You can extend it if you like.